src/physics/vakint.rs

```diff
@@ -18,7 +18,7 @@ fn vakint_to_python_error(vakint_error: VakintError) -> PyErr {
     exceptions::PyValueError::new_err(format!("Vakint error | {vakint_error}"))
 }
 
-#[pyclass(name = "Vakint", module = "symbolica", subclass)]
+#[pyclass(name = "Vakint", module = "symbolica",)]
```

Comment:
Why was the subclassing feature removed? There is a small chance people in Python want to subclass Vakint. IF removal is needed, the trailign comma can be removed too

Reply:
Ah yes, then it should be kept, I didn't know this was to allow subclassing.
Also, @benruijl does the module need to be `symbolica` here or `symbolica_community`.
I think it needs to be `symbolica_community` because otherwise it seems to give weird pathing for `Vakint` when getting it from `from symbolica_community *`.

Reply:
I think you are right, it should be symbolica_community. The same then holds for the spenso structs. I think it's nice for everyone to see that these additions are community made.

Reply:
Yes I had it set to symbolica_community.tensors on my en but I forgot to change it here

Reply:
Yes I had it set to symbolica_community.tensors on my end but I forgot to change it here